react-backend/client/src/components/visitors/homeVisitor.js

```diff
@@ -18,41 +69,54 @@ const postFeedback = (feedbackScore, props) => {
     }),
   }).then(() => props.history.push('/thankyou'));
 };
+
+const logout = props => () => {
+  localStorage.removeItem('token');
+  props.updateLoggedIn();
+};
+
 export default props => (
   <div>
-    <h1>
-      Welcome visitor! <br /> What do you want to do?<br />
-    </h1>
-    <Link to="/visitor/signup">
-      <button className="Button">Sign up</button>
-    </Link>
-    <br />
-    <Link to="/visitor/login">
-      <button className="Button">Login</button>
-    </Link>
-    <br />
-    <div className="feedback-container">
-      <button className="feedback-button" onClick={() => postFeedback(-1, props)}>
-        <img src={sad} alt="sad feedback button" />
-      </button>
-      <button className="feedback-button" onClick={() => postFeedback(0, props)}>
-        <img src={confused} alt="confused feedback button" />
-      </button>
-      <button className="feedback-button" onClick={() => postFeedback(+1, props)}>
-        <img src={happy} alt="happy feedback button" />
-      </button>
-    </div>
-    <br />
-    <Link to="/">
-      <button className="Button ButtonBack">Back</button>
-    </Link>
-    <br />
-
-    <a
-      href="http://www.powertochange.org.uk/data-protection-funding-applications/"
-      className="Policy"
-    >
+    <FlexContainerCol justify="space-around">
+      <StyledNav>
+        <FlexItem>
+          <HyperLink to="/">Back to the main page</HyperLink>
+        </FlexItem>
+        <FlexItem flex="2">
+          <Heading>Welcome Visitor</Heading>
+        </FlexItem>
+        <FlexItem>
+          <HyperLink to="/cb/login" onClick={logout(props)}>
+            Logout
+          </HyperLink>
+        </FlexItem>
+      </StyledNav>
+      <StyledSection>
+        <FlexLink to="/visitor/signup">
+          <ButtonLeft large>Sign up</ButtonLeft>
```

Comment:
In the design, the left button is the login button, and has the copy `Sign in with QR Code`.